adoc/deployment-aws.adoc

```diff
@@ -93,6 +93,174 @@ requires your credentials. These can be obtained by following these steps:
 * Click menu:Create Access Key[] on the "Security Credentials" tab.
 * Note down the newly created _Access_ and _Secret_ keys.
 
+=== Create IAM Role, Role Policy, and Instance Profile through AWS CLI ===
+
+For user who does not have permission to create IAM role, role policy, and instance profile using terraform, devops should create them for you using instruction below:
```

Comment:
`Users who do not have permission ...`
`... create them for you, using the instructions below:`

Reply:
changed